### DISCUSSION TITLE
Removing nodes in a distributed enviroment

### DISCUSSION THREAD

**Discussion by welch27330** [Security-Onion-Solutions/securityonion#3707] [Unsupported Versions] [answered]
I work in a dynamic and air-gapped environment. I was looking for a way to remove a sensor from the master after I stand one up and then take it back down so that it does not show in the grid. Currently on version 2.3.40.

**Comment by darth-drew** [comment, marked as answer]
They make that super easy.

https://docs.securityonion.net/en/2.3/removing-a-node.html#removing-a-node

**Reply by welch27330** [reply]
I guess reading the RTFM would have helped. Thank you.

**Reply by darth-drew** [reply]
Lol no worries mate. The docs aren't perfect, but they are pretty great. Sometimes the structure confuses me a bit so I use that search bar a lot, personally.

**Reply by welch27330** [reply]
I searched again because I am having an issue deploying a standalone FLEET node. The errors.log shows:
      [ERROR   ] 'mine.send': False
you have any tips for removing and then reinstalling a fleet node?

**Reply by darth-drew** [reply]
Recommend starting a new thread with a bit more background (environment, log, etc.). That way if I don't have an answer it'll be on an open thread.
